.circleci/config.yml

```diff
@@ -24,30 +24,44 @@ jobs:
       - checkout
 
       - run:
-         name: create virtualenv and install python dependencies
-         command: |
-           python3.6 -m venv legacy_code/venv
-           . legacy_code/venv/bin/activate
-           pip install -r legacy_code/requirements.txt
+          name: create virtualenv and install python dependencies
+          command: |
+            python3.6 -m venv legacy_code/venv
+            . legacy_code/venv/bin/activate
+            pip install -r legacy_code/requirements.txt
       - run:
-         name: Wait for DB
-         command: dockerize -wait tcp://localhost:5432 -timeout 1m
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:5432 -timeout 1m
 
-      - run: 
-         name: Install PostgreSQL Client
-         command: |
-           echo "deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
-           wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
-           sudo apt-get update
-           sudo apt-get install postgresql-client-9.5
+      - run:
+          name: Install PostgreSQL Client
+          command: |
+            echo "deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+            wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+            sudo apt-get update
+            sudo apt-get install postgresql-client-9.5
 
       - run:
-          name: tests
+          name: tests_legacy
           command: |
             . legacy_code/venv/bin/activate
             python legacy_code/tests/test_event_parser.py
             python legacy_code/tests/test_pageview_parser.py
             python legacy_code/tests/test_uploader.py
             python legacy_code/tests/test_phone_parser.py
-            
-            
+
+      # For login_analytics python library
+      - run:
+          name: create virtualenv and install python dependencies
+          command: |
+            python3.6 -m venv ./venv
+            source ./venv/bin/activate
+            pip install --upgrade pip
+            pip install .
+            pip install -rrequirements-test.txt
```

Comment:
should there be a space between `-r` and `requirements-test.txt`?

Reply:
No there is not, that is the syntax, I am pretty sure.

Reply:
I think the convention for any command line option is to have a space between the option and the argument. See the documentation: https://pip.pypa.io/en/stable/reference/pip_install/#options